Maven enforcer reports incorrect error message

It reports this error message when running with Apache Maven 3.5.2 and fails the build.
```
[INFO] --- maven-enforcer-plugin:3.0.0-M1:enforce (enforce-maven) @ vavr-parent ---
[WARNING] Rule 0: org.apache.maven.plugins.enforcer.RequireMavenVersion failed with message:
Maven 2.1.0 and 2.2.0 produce incorrect GPG signatures and checksums respectively.
```
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-enforcer-plugin:3.0.0-M1:enforce 
(enforce-maven) on project vavr-parent: Some Enforcer rules have failed. Look above for specific 
messages explaining why the rule failed. -> [Help 1]
```
But in fact it requires Maven 3.5.3.
https://github.com/vavr-io/vavr/blob/7506f7765abf88f5fb4c4ee9b78622dea8711af0/pom.xml#L122

--- a/pom.xml
+++ b/pom.xml
@@ -120,6 +120,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
                             <rules>
                                 <requireMavenVersion>
                                     <version>3.5.3</version>
+                                    <message>Minimum required Maven version to build vavr is 3.5.3</message>
                                 </requireMavenVersion>
                             </rules>
                         </configuration>

